bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fenecon"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="fenecon">
+
+		<label>FENECON Home</label>
+		<description>Thing for FENECON Home Device</description>
+		<category>Solarplant</category>
+
+		<channels>
+			<channel id="state" typeId="state"/>
+			<channel id="lastUpdate" typeId="lastUpdate"/>
+			<channel id="essSoc" typeId="essSoc"/>
+			<channel id="chargerPower" typeId="chargerPower"/>
+			<channel id="dischargerPower" typeId="dischargerPower"/>
+			<channel id="emergencyPowerMode" typeId="emergencyPowerMode"/>
+			<channel id="consumptionActivePower" typeId="consumptionActivePower"/>
+			<channel id="productionActivePower" typeId="productionActivePower"/>
+			<channel id="sellToGridPower" typeId="sellToGridPower"/>
+			<channel id="sellToGridEnergy" typeId="sellToGridEnergy"/>
+			<channel id="buyFromGridPower" typeId="buyFromGridPower"/>
+			<channel id="buyFromGridEnergy" typeId="buyFromGridEnergy"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the FENECON device, e.g. 192.168.1.11</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<label>Password</label>
+				<default>user</default>
+				<description>Password to access the device. The password for guest access is set by default.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="port" type="integer" min="1">
+				<context>network-address</context>
+				<label>Port</label>
+				<description>Port of the FENECON device</description>
+				<default>8084</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="1">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channel Types -->
+	<channel-type id="state">
+		<item-type>String</item-type>
+		<label>System State</label><!-- Use uppercase words, except prepositions. 2-3 words, max 25 chars -->
+		<description>FENECON system state: Ok, Info, Warning or Fault</description>
+		<category>Text</category>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="lastUpdate">
```

Comment:
See other comment, about naming convention, they also apply to channel type id's

Reply:
Done